examples/platform/nrfconnect/pw_sys_io/CMakeLists.txt

```diff
@@ -0,0 +1,21 @@
+include(chip-lib)
+include($ENV{PW_ROOT}/pw_build/pigweed.cmake)
+
+pw_add_module_library(pw_sys_io.nrfconnect
+    SOURCES
+      sys_io_nrfconnect.cc
+)
+
+target_include_directories(pw_sys_io.nrfconnect INTERFACE
+    $ENV{ZEPHYR_BASE}/include
+    include/
+)
+
+target_link_libraries(pw_sys_io.nrfconnect PUBLIC
+    zephyr_interface
+    pw_sys_io
+)
+
+target_compile_options(pw_sys_io.nrfconnect PUBLIC
+    -Wno-error=redundant-decls
```

Comment:
-Wno-redundant-decls

If it's not worth fixing, it's not worth seeing.

Reply:
They should be fixed, just not in this PR

Reply:
If it's not worth fixing in this PR, it's not worth seeing in this PR. Add a TODO.

Reply:
Without this compile option there are 20+ compilation failures like this:

```
/var/ncs/zephyr/include/arch/arm/aarch32/irq.h:40:13: error: redundant redeclaration of 'void arch_irq_enable(unsigned int)' in same scope [-Werror=redundant-decls]
   40 | extern void arch_irq_enable(unsigned int irq);
      |             ^~~~~~~~~~~~~~~
```

These are coming from nrfconnect. They should be fixed. The 'noise' of warnings produced may be a motivation to do so :)

Reply:
Filed https://github.com/project-chip/connectedhomeip/issues/4033